locale/import-symbols.py

```diff
@@ -65,81 +65,38 @@ def createCLDRAnnotationsDict(sources):
 	assert cldrDict, "cldrDict is empty"
 	return cldrDict
 
-# The list of languages supported by Speech-Dispatcher and the corresponding language(s) in the CLDR database
-# For language where there are no emojies, we put an empty list
-speechdToCLDRLocales = {
-	"an":(),
-	"ar":("ar",),
-	"bg":("bg",),
-	"ca":("ca",),
-	"cs":("cs",),
-	"da":("da",),
-	"de":("de",),
-	"el":("el",),
+# List of languages which are non-standard for CLDR files
+CLDRExceptionsList = {
 	"en":("en_001","en"),
-	"es":("es",),
-	"es_CO":(),
-	"fa":("fa",),
-	"fi":("fi",),
-	"fr":("fr",),
-	"ga":("ga",),
-	"gl":("gl",),
-	"he":("he",),
-	"hi":("hi",),
-	"hr":("hr",),
-	"hu":("hu",),
-	"is":("is",),
-	"it":("it",),
-	"ja":("ja",),
-	"ka":("ka",),
-	"kn":("kn",),
-	"ko":("ko",),
-	"lt":("lt",),
-	"my":("my",),
+	"es_US":("es_419","es_US"),
 	"nb_NO":("nb",),
```

Comment:
I don't think there is any reason to keep this `nb_NO` exception. Having `nb_NO` and no `nb` will be fine